From 7051f94aa3cd9dc4479eb43e292cf7fc483a5c1f Mon Sep 17 00:00:00 2001
From: "Patibandla, KiranX Kumar" <kiranx.kumar.patibandla@intel.com>
Date: Fri, 20 Nov 2020 13:29:08 +0530
Subject: [PATCH] Remove system writes vendor properties

-Property system_writes_vendor_properties_violators
-Postinstall does not access this property any more
Tracked-On: OAM-94404
Signed-off-by: Patibandla, KiranX Kumar <kiranx.kumar.patibandla@intel.com>

diff --git a/abota/generic/no_vendor_prefix/postinstall.te b/abota/generic/no_vendor_prefix/postinstall.te
deleted file mode 100644
index e26cac9..0000000
--- a/abota/generic/no_vendor_prefix/postinstall.te
+++ /dev/null
@@ -1,3 +0,0 @@
-typeattribute postinstall system_writes_vendor_properties_violators;
-set_prop(postinstall, ota_prop)
-
diff --git a/abota/generic/vendor_prefix/postinstall.te b/abota/generic/vendor_prefix/postinstall.te
deleted file mode 100644
index 333dad5..0000000
--- a/abota/generic/vendor_prefix/postinstall.te
+++ /dev/null
@@ -1,2 +0,0 @@
-typeattribute postinstall system_writes_vendor_properties_violators;
-set_prop(postinstall, vendor_ota_prop)
-- 
2.29.2

